Tests/SmokeDynamoDBTests/TypedDatabaseItem+RowWithItemVersionProtocolTests.swift

```diff
@@ -241,7 +241,8 @@ class TypedDatabaseItemRowWithItemVersionProtocolTests: XCTestCase {
         let pathMap = differences.pathMap
 
         XCTAssertEqual(pathMap["\"theList\"[1]"], .update(path: "\"theList\"[1]", value: "'eigthly'"))
-        XCTAssertEqual(pathMap["\"theList\""], .listAppend(path: "\"theList\"", value: "['ninthly', 'tenthly']"))
+        XCTAssertEqual(pathMap["\"theList\"[2]"], .update(path: "\"theList\"[2]", value: "'ninthly'"))
+        XCTAssertEqual(pathMap["\"theList\"[3]"], .update(path: "\"theList\"[3]", value: "'tenthly'"))
```

Comment:
Have you checked if this is accepted by DDB? These indexes do not exist.

Reply:
Clarified. All good.

Reply:
I tested it in beta dynmaodb perviously . If the index doesn’t exist, it will append the item. Similarly you can SET X.Y = Z even if X is not existing. Dynamodb will create X mapping with Y = Z 

reference: https://stackoverflow.com/questions/66534235/invalid-updateexpression-two-document-paths-overlap-with-each-other-must-remov/66547193#66547193